### DISCUSSION TITLE
Tutorial Error: "Module not found: Can't resolve "vocab_web"

### DISCUSSION THREAD

**Discussion by AustinThoughtLab** [l3atbc-datadog/pushkin#165] [General]
Currently working on the tutorial install and getting an error page when I try to navigate to http://localhost
I'm using Mac OS, and I couldn't find any mistakes in the code or directory. Any help is greatly appreciated!

Attached: Error message photo that reads "Failed to compile. './src/experiments.js' Module not found: Can't resolve 'vocal_web' in '/usr/src/app/src'.

![Screen Shot 2020-09-02 at 2 31 36 PM](https://user-images.githubusercontent.com/53018617/92028048-06b21e80-ed29-11ea-88a7-8a777a250d90.png)

**Comment by parkerrobbins** [comment]
Hi! Thanks for reaching out! A couple of questions:
1. Could you tell me what version of Pushkin you're running (`$ pushkin --version`)?
2. Are you using the `default` or `basic` site template?
3. If you're following the Quickstart, have you also installed a `mind` and a `whichenglish` experiment using the basic experiment template?

**Reply by AustinThoughtLab** [reply]
Thanks for the quick response! 

1. Version 2.2.10
2. Default
3. I have also installed mind and whichenglish, and their file directories seem to match vocab.

Thanks again!

**Reply by parkerrobbins** [reply]
Thanks for that! 

Did you install the experiments with version 2.2.10 or did you install them with a previous version and then upgrade to 2.2.10? 

If the latter, you may need to complete the migration instructions as described in step 3 [here](https://github.com/pushkin-consortium/pushkin/discussions/138#discussioncomment-58671).

**Reply by AustinThoughtLab** [reply]
I installed the experiments with version 2.2.10. I did have to restart the process halfway through using the kill command to get rid of some other errors, but this one persists. Could the problem be related to stopping and starting over?

**Reply by parkerrobbins** [reply]
Possibly. I would suggest running `pushkin armageddon` and `pushkin prep` again before running `pushkin start;`. Have you tried that?

**Reply by restmegan** [reply]
Hi @AustinThoughtLab ! We will be working on another project this week and we hope to return to working on Pushkin issues on Friday, so if this is still an issue for you let us know and we will help you then.

**Reply by jkhartshorne** [reply]
@AustinThoughtLab -- apologies for the long delay. So what that sounds like is that at some point you installed an experiment called "vocab_web", and it got added to your `experiments.js` file. However, you don't actually have a "vocab_web" experiment. If you check your `experiments` folder, there probably isn't a subfolder called "vocab_web".

Or possibly I'm wrong. It would help if you could list the subfolders in "experiments" and also paste in the content of `experiments.js`.
